doc/source/whatsnew/v0.17.0.txt

```diff
@@ -429,6 +429,34 @@ Other API Changes
    ``raise ValueError``                All other public methods (names not beginning with underscores)
    ===============================     ===============================================================
 
+- Clarifying ``display.precision`` option to refer to decimal places (:issue:`10451`)
```

Comment:
make this a separate sub-section (e.g. use the ``^^^`` to outline the header). (and move before Other API Changes)